test/system/254-podman-quadlet-multi.bats

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env bats   -*- bats -*-
```

Comment:
None of the test use additional systemd sections. Need to make sure that these section both don't interfere with the creation of the standalone files and that they are copied correctly

Reply:
Done, added in first test.